app/containers/message/Components/Attachments/Attachments.tsx

```diff
@@ -77,16 +76,7 @@ const Attachments: React.FC<IMessageAttachments> = React.memo(
 				return <CollapsibleQuote key={index} attachment={file} timeFormat={timeFormat} getCustomEmoji={getCustomEmoji} />;
 			}
 
-			return (
-				<Reply
-					key={index}
-					attachment={file}
-					timeFormat={timeFormat}
-					getCustomEmoji={getCustomEmoji}
-					msg={msg}
-					showAttachment={showAttachment}
-				/>
-			);
+			return null;
 		});
 		return <View style={{ gap: 4 }}>{attachmentsElements}</View>;
```

Comment:
If there's no `attachmentsElements`, we should not return an empty view with a gap of 4.